### PR TITLE
docs: Adding troubleshooting section for WSL2

### PR DESCRIPTION
**Pull request by PatrickLang**
Just linking workarounds for a few known issues. I'm not sure if/when they'll be fixed since both are related to systemd and there doesn't seem to be a motion to enable that on WSL2 yet.

### CODE REVIEW

File: site/content/docs/user/using-wsl2.md [hunk 1]
```diff
@@ -66,6 +66,48 @@ Once your Windows Insider machine is ready, you need to do a few more steps to s
 
 Now, move on to the [Quick Start](/docs/user/quick-start) to set up your cluster with kind.
 
+## Troubleshooting
+
+### Failures with `kind cluster create`
```
Comment (aojea) [inline site/content/docs/user/using-wsl2.md]: this renders a bit weird to me, is this on purpose using the code marks on the title?

Comment (BenTheElder) [reply site/content/docs/user/using-wsl2.md]: https://deploy-preview-984--k8s-kind.netlify.com/docs/user/using-wsl2/#troubleshooting

**Comment by BenTheElder** [comment]
/lgtm
/approve
thank you!

File: site/content/docs/user/using-wsl2.md (hunk 1 above)
Comment (aojea) [reply site/content/docs/user/using-wsl2.md]: :man_facepalming: 
I always forget I can see the preview on that link
sorry

**Pull request merged**